### PR TITLE
accessibility: issue/#2165 updated component description

### PR DESCRIPTION
**Pull request by oliverfoster**
[#2165](https://github.com/adaptlearning/adapt_framework/issues/2165)
* Made component description easier to digest

### CODE REVIEW

File: properties.schema [hunk 1]
```diff
@@ -7,15 +7,7 @@
...
-      "default": "This is a media player component. Select the play / pause button to watch or listen.",
-      "inputType": "Text",
-      "validators": [],
-      "translatable": true
-    },
-    "transcriptButton": {
-      "type": "string",
-      "required": false,
-      "default": "Alternatively you can select the link below for the transcript.",
+      "default": "Video{{#any _transcript._inlineTranscript _transcript._externalTranscript}} and transcript.{{else}}.{{/any}}",
```
Comment (moloko) [inline properties.schema]: Wouldn't 
```json
"Video{{#any _transcript._inlineTranscript _transcript._externalTranscript}} and transcript{{/any}}."
```
give you the same result?

**Comment by tomgreenfield** [comment]
How about if someone uses the audio-only player?

**Comment by moloko** [comment]
How about "Media player with transcript."?

In an ideal world you'd have "Media player with transcript and closed-captions." but I suspect that could start to get quite complicated..!

**Comment by oliverfoster** [comment]
Lol, I'll change to Media player rather than video.

We don't need to mention the captions to the screen reader user.
| Tool | Visible | Audible |
| ---- | ---- | ---- |
| Screen reader | n | y |
| Captions | y | n |

**Comment by oliverfoster** [comment]
closed in favour of https://github.com/adaptlearning/adapt-contrib-media/pull/183

**Pull request closed without merging**